src/client/pfs/pfs.proto

```diff
@@ -561,6 +561,16 @@ message DiffFileResponseV2 {
   FileInfo new_file = 2;
 }
 
+message CreateTempFileSetResponse {
```

Comment:
Just a nitpicky thing, but I have been using `tmp` as a shorthand for temporary and it would be nice to be consistent about this in PFS / PPS.